src/compile/model.ts

```diff
@@ -312,6 +355,19 @@ export abstract class Model {
     return this._legend[channel];
   }
 
+  public title(channel: Channel) {
```

Comment:
This method is like `field()`, which is based on the assumption that there is one field per channel, add thus will not work with multiple fields due to production rules.

Will note this in #1263.